force-app/main/default/classes/FlowControlLoopsCollections.cls

```diff
@@ -183,14 +259,17 @@ public with sharing class FlowControlLoopsCollections {
      */
     public static List<Integer> createAndPopulateList() {
         // Create a new list of integers
-
+        List<Integer> numbers = new List<Integer>();
+        numbers.add(1);
+        numbers.add(2);
+        numbers.add(3);
         // Add the integers 1, 2, and 3 to the list
         // add 1
         // add 2
         // add 3
 
         // Return the populated list
-        return null; // Replace null with the variable you used to store the result
+        return numbers; // Replace null with the variable you used to store the result
```

Comment:
Good work. Senior dev tip here is to use the List contractor when you know all the values at right time. 

```
 public static List<Integer> createAndPopulateList() {
      return new List<Integer>{1, 2, 3};
    }
```